blazor/file-manager/data-binding.md

```diff
@@ -1,13 +1,255 @@
 ---
 layout: post
-title: Flat data rendering in Blazor FileManager component | Syncfusion
-description: Checkout and learn here all about flat data rendering in Syncfusion Blazor FileManager component and more.
+title: Data Binding in Blazor FileManager Component | Syncfusion
+description: Checkout and learn here all about Data Binding in Syncfusion Blazor FileManager component and much more.
 platform: Blazor
 control: File Manager
 documentation: ug
 ---
 
-# Flat data rendering in Blazor FileManager component
+# Data Binding in Blazor FileManager Component
+
+The FileManager uses [SfDataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.SfFileManager-1.html), which supports both RESTful JSON data services binding and IEnumerable binding. It provides the option to load data either with the [AjaxSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerAjaxSettings.html) property or list of objects by providing the response within the corresponding events.
```

Comment:
Whether File Manager use SfDataManager?

Reply:
The FileManager component does not use SfDataManager. I think instead of SfFileManager, he used SfDataManager.